releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -2,6 +2,11 @@
 
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v1.6.4 - June, 19 2019
+* Improved [homepage documentation](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task):
+  * Added a link to the release notes.
```

Comment:
Will this avoid some confusion we are referring to some other release notes?
*Added a link to **this** release notes.*

Reply:
Changed to `* Added a link to this release notes.`